change to object and added note about capitalization (#497)
medic/cht-docs: content/en/apps/reference/app-settings/forms.md

keywords: workflows interop json-forms
---

> JSON forms are defined in either the `base_settings.json` or the `app_settings/forms.json` file and compiled in to the `app_settings.json` file with the `compile-app-settings` action in the `medic-conf` tool. JSON Forms are used for parsing reports from formatted SMS, SIM applications, and Medic Collect. JSON form definitions are also used for interoperability with third-party systems. Each form is defined as an JSON form object according to the following schema. The key for each object is required to be capitalized. 

Reviewer: Would be good to clarify that it is the entire string that needs to be uppercase, and I also think uppercase will reasonate more than capitalized to say that. 

```suggestion
JSON forms are defined in either the `base_settings.json` or the `app_settings/forms.json` file and compiled in to the `app_settings.json` file with the `compile-app-settings` action in the `medic-conf` tool. JSON Forms are used for parsing reports from formatted SMS, SIM applications, and Medic Collect. JSON form definitions are also used for interoperability with third-party systems. Each form is defined as an JSON form object according to the following schema. The key for each object must be unique and all characters must be uppercase. 
```
